packages/core/core/src/middlewares/security.ts

```diff
@@ -30,24 +31,45 @@ const defaults: Config = {
 };
 
 export const security: Common.MiddlewareFactory<Config> =
-  (config, { strapi }) =>
-  (ctx, next) => {
+  (config: any, { strapi }: { strapi: Strapi }) =>
```

Comment:
Why do you need to add this? It should be inferred from the MiddlewareFactory type